cpp/ps0001/ps0001cpp.cpp

```diff
@@ -1,21 +1,17 @@
-/******************************************************************************
-
-Welcome to GDB Online.
-GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
-C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
-Code, Compile, Run and Debug online from anywhere in world.
-
-*******************************************************************************/
 #include <iostream>
 
 using namespace std;
 int main()
 {
-    int a,b,c,result;
-    cout <<"enter three numbers to add \n";
-    cin>>a>>b>>c;
-    result=a+b+c;
-    cout<<"the sum of a , b and c =" << result ;
+    // prompting users for 3 numbers  1/2
+    unsigned short a, b, c, result;
+    cout << "enter three numbers to add \n";
+    cin >> a >> b >> c;
+
+    // outputting the result of the summation process 3/3
+    result = a + b + c;
+    cout << "the sum of a , b and c =" << result ;
```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as program is user-friendly